### ISSUE TITLE
Bolder and bigger text for the street name in navigation

### ISSUE THREAD

**Issue by biodranik** [organicmaps/organicmaps#5316]
Users want a better visible next street name.

![image](https://github.com/organicmaps/organicmaps/assets/170263/283ce86d-2b7c-435d-a9de-81a79d37a32d)

**Comment by fitojb** [comment]
Making it similar to a guide sign (white-on-green instead of black-on-white, a bolder font more reminiscent of street signs...) would make it more obvious.

**Comment by uraj323** [comment]
Can you assign this issue to me?

**Comment by biodranik** [comment]
Nope. We don't assign issues to newcomers. You can safely start working on any issue and make a pull request with the fix instead.

**Comment by Jean-BaptisteC** [comment]
Similar to #2177
